### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by Maxi2022gt** [im557/psych-discussions-alternative#12] [General]
why now psych discussions is gone?
and the readme is just mickey mouse?
![why](https://user-images.githubusercontent.com/106933671/176738374-d9293964-0827-42bd-8323-a9ed6260b7f6.png)
edit:oh and i was from the psych dicussions

**Comment by im557** [comment]
i have no idea lol

**Reply by Maxi2022gt** [reply]
yeah, its kinda weird
